psamm_import/importer.py

```diff
@@ -124,6 +124,28 @@ def decimal_representer(dumper, data):
     return dumper.represent_scalar('tag:yaml.org,2002:float', value)
 
 
+def detect_extracellular(model):
+    """Detect the identifier for equations with extracellular compartments.
+    """
+    extracellular_key = Counter()
+
+    for reaction_id, reaction in iteritems(model.reactions):
+        if 'equation' not in reaction.properties:
+            continue
+
+        equation = reaction.properties['equation']
+        if len(equation.compounds) == 1:
+            compound, _ = equation.compounds[0]
+            compartment = compound.compartment
+            extracellular_key[compartment] += 1
+    if len(extracellular_key) == 0:
+        best_key = None
```

Comment:
I would change the last line to `return None` so we can avoid printing the log message in this case. Also missing space between `)` and `==`.